controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/TenantSerializer.java

```diff
@@ -145,6 +147,67 @@ private CloudTenant cloudTenantFrom(Inspector tenantObject) {
         return keys.build();
     }
 
+    private TenantInfo tenantInfoFromSlime(Inspector infoObject) {
+        return TenantInfo.EmptyInfo
```

Comment:
This chould just call the constructor directly as the immutable builder methods will create a new object per field.

Reply:
Not sure I understand - what is the difference?

Reply:
The difference is the number of objects you create. In the current state its is multiplied by the number of fields in `TenantInfo` (since each `withXXX` creates a new object), this is then multiplied by the number of tenants you're deserializing.

Reply:
Ah - an optimisation. Using builder methods adds some benefits that I think outweigh that - one is that the order of the constructor when using strings are prone to mistakes (and this eliminates that).